src/main/scala/mvp2/actors/Publisher.scala

```diff
@@ -45,24 +52,31 @@ class Publisher(settings: Settings) extends CommonActor {
       networker ! keyBlock
       lastKeyBlock = keyBlock
       mempool.removeUsedTxs(keyBlock.transactions)
-    case Get =>
-      val newBlock: KeyBlock = createKeyBlock
+    case RequestForNewBlock(isFirstBlock, schedule) =>
```

Comment:
seems this is not necessarily to publish block
case RequestForNewBlock(isFirstBlock, schedule)